fapolicy_analyzer/ui/strings.py

```diff
@@ -188,3 +188,6 @@
 )
 
 APPLY_CHANGESETS_ERROR_MESSAGE = _("Error applying changes")
+
+TIME_SELECTION_START_INVALID = _("Selected Start Time is not a valid date")
+TIME_SELECTION_STOP_INVALID = _("Selected Stop Time is not a valid date")
```

Comment:
You'll need to run the `extract_messages` script an commit the updated .pot file with the new strings.